SpatialGDK/Source/SpatialGDK/Public/Utils/WorkerVersionCheck.h

```diff
@@ -4,7 +4,7 @@
 
 #include "improbable/c_worker.h"
 
-#define WORKER_SDK_VERSION "15.3.1"
+#define WORKER_SDK_VERSION "15.4.0"
```

Comment:
Since we are using custom only versions from now on, I think we should start checking `WORKER_API_VERSION_FULL` instead. See:
https://improbableio.atlassian.net/browse/UNR-5907

Reply:
Good point - I'll sync with runtime team to check how we want to manage this going forward

Reply:
@m-samiec checkout the ticket and the slack discussion in that ticket, I started a lot of that discussion already, you may be able to get what you need from there, if not at least some useful context.